canonical-web-and-design/ubuntu.com
# Add notification banner and form for kernel beta signup

## Done

- Added notification banner linking to kernel signup form to:
  - `/telco`
  - `/iot`
  - `/robotics`
  - `/embedded`
- Built kernel beta signup form based on [the brief](https://docs.google.com/document/d/1agEGZVSXqhaffajJpnNCgudG9_CBz1rE2vjZ2eHP7d0/edit#)
- Added redirect from ubuntu.com/realtime-kernel to /embedded#beta-signup

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: 
    - http://0.0.0.0:8001/telco
    - http://0.0.0.0:8001/iot
    - http://0.0.0.0:8001/robotics
    - http://0.0.0.0:8001/embedded
    - Be sure to test on mobile, tablet and desktop screen sizes
- See that banner renders correctly on above pages
- Test form 

## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/4972
